extension/main.js

```diff
@@ -13,6 +13,15 @@
 		chrome.browserAction.setTitle({
 			title: title
 		});
+
+		if (badge && window.GitHubNotify.settings.get('showDesktopNotif')) {
+			chrome.notifications.create('github-notifier', {
+				title: 'GitHub',
+				iconUrl: 'icon-128.png',
+				type: 'basic',
+				message: 'You have new unread notifications'
```

Comment:
@inikulin I'm really excited about this PR getting merged in, however I think I agree with @sindresorhus that it would make it a lot more useful if you could include the title or at least the repository the notification is from. I don't think the argument is about if this should be a viewer or notifier it's about giving the user enough info so they can make a decision on whether or not go to github to view the message. I get tons of notifications from different repos and it would help me a lot if I could tell where the notification is from, if I'm at work and it's a work repo then I should probably check it, but if it's from one of my hobby projects it's ok to ignore.